### ISSUE TITLE
SPIKE: Determine the approach for refresh token scenario

### ISSUE THREAD

**Issue by kami619** [keycloak/keycloak-benchmark#538]
### Description

Possible approaches

- One way could be to introduce a new cli argument to the kcb.sh script to dynamically `count` the total number of logout requests in order to be able to populate `x` number of sessions before we start refreshing the sessions using token refresh endpoint.
- Another approach could be to, create a set number of sessions and then re-use those session tokens and refresh them instead of creating new tokens all the time, to make it more practical load simulation.

**Comment by ahus1** [comment]
Obsolete due to #518. Will open a new issue if more needs would arise.
